src/post.js

```diff
@@ -24,17 +24,19 @@ module.exports = function (config, cb) {
       if (err) return cb(err)
 
       var ghRepo = parseSlug(pkg.repository.url)
+      var tagName = (pkg.publishConfig || {}).tag || 'v' + pkg.version
+
       var tag = {
         owner: ghRepo[0],
         repo: ghRepo[1],
-        ref: 'refs/tags/v' + pkg.version,
```

Comment:
now you are replacing creating the `v1.2.3` tags. We will not remove that feature. But we could additionally create a tag if a custom tag is set in `publishConfig`.

Reply:
Not sure I understand the first part. Shouldn't it match with release.tag_name and release.name?

Reply:
instead of creating a single tag, we know would create 2 tags at once:

1. the version tag in the form of `v1.2.3`
2. the release tag which matches the name of the tag used to publish the package on npm

Does that make sense?

Reply:
ah, so this line would be kept in v1.2.3 format and only the release one changes if publishConfig is present?

Reply:
nothing changes, if we release a new version `1.2.3` to npm, we will still create a git tag `v1.2.3`. But if `publishConfig.tag` is to set `"next"`  then we can create an _additional_ git tag called `next`. 

What I’m not exactly sure about is what happens if a git tag `next` already exists from an earlier release. We would probably check if it does, if yes delete it, then create it again.

Reply:
So I just need to create an additional reference (github.gitdata.createReference) when publishConfig.tag is present, right?

Reply:
yes :)